compiler/luci/service/src/ShapeInfer_StridedSlice.cpp

```diff
@@ -433,7 +433,7 @@ loco::TensorShape infer_output_shape(const CircleStridedSlice *node)
   for (uint32_t idx = 0; idx < shape_size; ++idx)
   {
     // reverse copy
-    output_shape.dim(idx) = output_shape_vector.at(shape_size - 1u - idx);
+    output_shape.dim(idx) = static_cast<uint32_t>(output_shape_vector.at(shape_size - 1u - idx));
```

Comment:
Does our SE tool accepts this pattern? I think downcasting would need range check?